src/classes/ADDR_AdminAcc_TEST.cls

```diff
@@ -0,0 +1,891 @@
+            acc.BillingStreet = 'new street\r\nsecond line';
+            acc.BillingCity = 'new city';
+        }
+        Test.startTest();
+        insert accs; 
+        Test.stopTest();       
+        
+        // verify the Address objects split street.
+        list<Address__c> listAddr = [select Id, MailingStreet__c, MailingStreet2__c, Formula_MailingStreetAddress__c from Address__c];
+        System.assertEquals(2, listAddr.size());
+        for (Address__c addr : listAddr) {
+            system.assertEquals('new street', addr.MailingStreet__c);
+            system.assertEquals('second line', addr.MailingStreet2__c);
+            system.assertEquals('new street, second line', addr.Formula_MailingStreetAddress__c);
+        }
+
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
+        BillingPostalCode, BillingCountry, BillingLatitude, BillingLongitude, is_Address_Override__c, Current_Address__c from Account]);
+        
+        for (Account acc : mapAccIdAcc.values()) {
+            system.assertEquals('new street\r\nsecond line', acc.BillingStreet);
+            system.assertEquals(false, acc.is_Address_Override__c);
+            system.assertNotEquals(null, acc.Current_Address__c);
+        }
+    }
+
+    /*********************************************************************************************************
+    @description Insert new default multiline street addresses w/ existing default addresses  
+    verify:
+        Account address matches new default
+        street & street2 combined into contact and account single street field
+    **********************************************************************************************************/            
+    @isTest
+    public static void insertNewDefaultMultilineStreetAddr() {
+        configSettings();
+             
+        // this creates a default Address for each Account
+        UTIL_UnitTestData_TEST.AccsWithAddrs accsAddrs = UTIL_UnitTestData_TEST.createTestAccsAddrs(2, UTIL_Describe.getAdminAccRecTypeID());
+        List<Address__c> addrs = accsAddrs.addrs;
+        
+        // create additional addresses
+        addrs = UTIL_UnitTestData_TEST.initTestAddr(2);
+        for (Integer i = 0; i < accsAddrs.accs.size(); i++) {
+            addrs[i].Parent_Account__c = accsAddrs.accs[i].Id;
+            addrs[i].Default_Address__c = true;
+            addrs[i].MailingStreet__c = 'New Default Street';
+            addrs[i].MailingStreet2__c = 'Second Line';
+            addrs[i].MailingCity__c = 'New Default City' + i;
+        }
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        Test.startTest();
+        insert addrs;
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;        
+        Test.stopTest();
+
+        // verify Account address is new
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. I suggest to introduce a `WHERE` clause or cap the max number of records returned adding a `LIMIT` clause to the query.